portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,347 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+/* For all pages */ 
+
+html {
+  height: 100%;
+  margin: 0;  
+  scroll-behavior: smooth;
+}
+
+body {
+  font-family: 'Work Sans';
+  height: 100%;
+  margin: 0;
+}
+
+h1 {
+  font-size: 60px;
+  font-weight: bold;
+}
+
+h2 {
+  font-size: 30px;
+}
+
+p {
+  font-size: 20px;
+}
+
+img {
+  max-height: 100%;
+  max-width: 100%;
+}
+
+a {
+  text-decoration: none;
+}
+
+.topnav {
+  background: lavender;
+  overflow: hidden;
+  padding: 16px;
+}
+
+.topnav a {
+  color: black;
+  display: block;
+  float: right;
+  font-size: 20px;
+  padding: 14px 16px;
+  text-align: center;
+  text-decoration: none;
+}
+
+.topnav-left a {
+  color: black;
+  display: block;
+  float: left;
+  font-size: 20px;
+  padding: 14px 16px;
+  text-align: center;
+  text-decoration: none;
+}
+
+.topnav a:hover {
+  color: #8e8ead;
+}
+
+#to-top-btn {
+  background: #cec6e3;
+  border: none;
+  border-radius: 10px; /* Round corners */
+  bottom: 50px;
+  color: black;
+  cursor: pointer; /* Add mouse pointer on hover */
+  outline: none;
+  padding: 10px 16px 0;
+  position: fixed;
+  right: 50px;
+  font-size: 25px;
+  z-index: 99; /* Won't overlap */
+}
+
+#to-top-btn:hover {
+  background: #8e8ead;
+}
+
+#pad {
+  margin: 0;
+  padding: 80px 140px 20px;
+}
+
+@media screen and (max-width: 600px) {
```

Comment:
I would usually add a comment here describing what the css controls. It seems less straightforward what this is accomplishing.